helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data>
+    <record model="ir.module.category" id="module_helpdesk_category">
+      <field name="name">Helpdesk</field>
+      <field name="description">Helps you handle your helpdesk security.</field>
+      <field name="sequence">9</field>
+    </record>
+  </data>
+  <data noupdate="1">
+    <!--Email template -->
+    <record id="assignment_email_template" model="mail.template">
+      <field name="name">Ticket Assignment</field>
+      <field name="model_id" ref="model_helpdesk_ticket"/>
+      <field name="email_from">${object.company_id.partner_id.email}</field>
+      <field name="subject">${object.company_id.name} Payment Receipt (Ref ${object.name or 'n/a' })</field>
```

Comment:
this "Payment Receipt" subject is also present in version 11.0 but seems a bad left over from som copied code. Changing it would be great.

Reply:
@rvalyi Please create a new PR to change that in v11